contracts/truefi/TrueFiPool.sol

```diff
@@ -649,6 +667,21 @@ contract TrueFiPool is ITrueFiPool, ERC20, ReentrancyGuard, Ownable {
         emit Collected(beneficiary, amount);
     }
 
+    function sellCrvWith1Inch(bytes calldata data) external onlyOwnerOrManager {
+        (address caller, I1Inch.SwapDescription memory description, ) = abi.decode(
+            data[4:],
+            (address, I1Inch.SwapDescription, I1Inch.CallDescription[])
+        );
+        require(caller == 0xe069CB01D06bA617bCDf789bf2ff0D5E5ca20C71, "TrueFiPool: Unknown caller");
+        require(description.srcToken == address(_minter.token()), "TrueFiPool: Source token is not CRV");
+        require(description.dstToken == address(_currencyToken), "TrueFiPool: Destination token is not TUSD");
+        require(description.dstReceiver == address(this), "TrueFiPool: Receiver is not pool");
+
+        _minter.token().approve(address(_1inchExchange), description.amount);
+        (bool success, ) = address(_1inchExchange).call(data);
+        require(success, "TrueFiPool: 1Inch swap failed");
```

Comment:
consider checking balances after to make sure we get enough back
set max slippage at 1%